scripts/triage

```diff
@@ -0,0 +1,17 @@
+#!/usr/bin/python
+
+from github import Github
+import sys
+import os
+
+query = sys.argv[1]
+
+github = Github(os.environ['GITHUB_TOKEN'])
+org = github.get_organization('enarx')
+project = [p for p in org.get_projects(state='open') if p.name == 'Planning'][0]
+column = [c for c in project.get_columns() if c.name == 'Triage'][0]
+
+for issue in github.search_issues(f"{query} state:open is:public -project:enarx/{project.number}"):
```

Comment:
1) If we have two projects (planning and sprint), then the bot should check if issues are not in either, instead of not in just Planning. If an issue is in the Sprint board but not in Planning, this will dump it in Triage when it shouldn't be.
2) This search queries for organization using input to the script, but it hardcodes organization when looking for projects to exclude from. This could make it hard to test outside the main Enarx organization. But if that's not a priority, then not a big deal.

Reply:
@mbestavros 

1. It is my plan to keep all issues in the `Planning` board always. They will be additionally added to the `Sprint` board during a `Sprint`.

2. Will fix.